src/test/meson.build

```diff
@@ -259,6 +264,7 @@ executables += [
                         libbasic_compress,
                         libshared,
                 ],
+                'dependencies' : compression_libs,
```

Comment:
frankly, why not switch this over too? I mean, it's good testing dlopen() properly